docs/requirements.txt

```diff
@@ -0,0 +1,4 @@
+sphinx
+paramiko
```

Comment:
Maybe `keg_storage[sftp]` instead? 

We need the pip `extras` so that `autodoc` works.